.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-3.2.2
```

Comment:
ruby version is specified at various places in the app. what are those? and what happens if I change it at one place and don't change it at other?

Reply:
Ruby Version in .ruby-version tells what version of ruby application should use, if we change it then it will give an error that the version is not equal to the installed ruby version.

Ruby Version in Gemfile helps in installing gems compatible to the mentioned version of ruby.

If we change the version at anyplace, it will give error ```Your Ruby version is 3.2.2, but your Gemfile specified 3.1.3 ```

Reply:
If we change the version in Gemlock file nothing will happen